nextflow_schema.json

```diff
@@ -238,24 +238,38 @@
                     "description": "Turns on long read quality control steps (adapter clipping, length filtering etc.)",
                     "help_text": "Turns on long read quality control steps (adapter clipping, length and/or quality filtering.)\n\nRemoving adapters (if present) is recommend to reduce false-postive hits that may occur from 'dirty' or 'contaminated' reference genomes in a profiling database that contain accidentially incorporated adapter sequences.\n\nLength filtering, and quality filtering can speed up alignment by reducing the number of unspecific reads that need to be aligned."
                 },
+                "longread_adapterremoval_tool": {
+                    "type": "string",
+                    "default": "porechop",
+                    "enum": ["porechop"],
+                    "fa_icon": "fas fa-hammer",
+                    "description": "Specify which tool to use for adapter trimming. Currently only one option."
+                },
                 "longread_qc_skipadaptertrim": {
                     "type": "boolean",
                     "description": "Skip long-read trimming",
                     "fa_icon": "fas fa-forward",
                     "help_text": "Skip removal of adapters by Porechop. This can be useful in some cases to speed up run time - particularly when you are running data downloading from public databases such as the ENA/SRA that should already have adapters removed. We recommend that you check your FastQC results this is indeed the case."
                 },
+                "longread_filter_tool": {
+                    "type": "string",
+                    "default": "filtlong",
+                    "enum": ["filtlong", "nanoq"],
+                    "fa_icon": "fas fa-hammer",
+                    "description": "Specify which tool to use for long reads filtering"
```

Comment:
Can you give any guidance in teh long-form help text which to use when?

Reply:
Done! I also set the default as `nanoq`